my-vault/internal/handlers/handlers.go

```diff
@@ -0,0 +1,205 @@
+package handlers
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"log/slog"
+	"net/http"
+	"os"
+
+	"github.com/bmasaiti/go-projects/my-vault/internal/domain"
+	"github.com/bmasaiti/go-projects/my-vault/internal/storage"
+	"github.com/google/uuid"
+)
+
+var logger = slog.New(slog.NewJSONHandler(os.Stdout, nil))
```

Comment:
I think we should pass this in from `main.go` instead of creating it as a global variable. You could add it to the `SecretHandler` struct instead. What do you think?